scripts/install/roles/env/tasks/main.yml

```diff
@@ -16,7 +16,7 @@
 - name: "Install new version (zsh)"
   lineinfile:
     dest: "{{ ansible_env.HOME }}/.zshrc"
-    line: "source /opt/aeriscloud/scripts/env.sh"
+    line: "source {{ install_dir }}/scripts/env.sh"
```

Comment:
The playbook is always run by the install script which specifies the extra var, therefore it should be ok.